### ISSUE TITLE
Typing closing bracket of tags

### ISSUE THREAD

**Issue by trafnar** [imba/vscode-imba#82]
In vscode, typical behavior for a closing bracket/brace/paren is to type over it, so if you type `foo(` vscode might autocomplete to `foo()` placing the final `)` after your insertion point. When you then type a redundant `)` it replaces the existing one, rather than adding another.

I'd love to have that behavior when typing tags in imba. Currently, I type `<` and vscode adds the closing bracket so I have `<>` then I type the tag name: `<Foo>` and then I want to type the closing `>` which ends up as: `<Foo>>` rather than `<Foo>`.

![Screen Recording (Code)](https://user-images.githubusercontent.com/6732/106935782-a86eda80-66d0-11eb-80f3-1c4f15f1248a.gif)

**Comment by somebee** [comment]
Good idea. Just allowing the `<>` to autopair in the tag context but not when writing `if 100 <` etc was a major hassle and uses vscode api in a pretty hacky way – will see if this can be done using similar hacks :P
